src/Carbon/CarbonInterval.php

```diff
@@ -1435,11 +1435,9 @@ protected function getForHumansParameters($syntax = null, $short = false, $parts
             ];
         }
 
-        if ($altNumbers) {
-            if ($altNumbers !== true) {
-                $language = new Language($this->locale);
-                $altNumbers = \in_array($language->getCode(), (array) $altNumbers, true);
-            }
+        if ($altNumbers !== true) {
```

Comment:
```suggestion
        if ($altNumbers && $altNumbers !== true) {
```
Normally it's not equivalent if we allow anything not `true` instead of any truthy value except `true`.

Tests did not fail by luck, because if it's false/null/[] then \in_array() will return `false` as a language code string will never match. Still we execute some code and assignation that are not needed in this case.